pkg/bundle/internal/target/target.go

```diff
@@ -42,6 +42,11 @@ import (
 	"github.com/cert-manager/trust-manager/pkg/util"
 )
 
+const (
+	JksPasswdHashAnnotation    = "trust.cert-manager.io/jks-pwd-hash"
+	Pkcs12PasswdHashAnnotation = "trust.cert-manager.io/pksc12-pwd-hash"
+)
```

Comment:
For consistency, I think these annotations should be alongside the related data hash:

https://github.com/cert-manager/trust-manager/blob/17561571e008ac2775df6ef966bf89dd325a9913/pkg/apis/trust/v1alpha1/types_bundle.go#L26

Reply:
I agree. Fixed.